docs/static/includes/interfaces/int.cmk.udt.schema.bicep

```
@@ -29,8 +29,8 @@ resource cMKUserAssignedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentiti
   scope: resourceGroup(split((customerManagedKey.?userAssignedIdentityResourceId ?? '//'), '/')[2], split((customerManagedKey.?userAssignedIdentityResourceId ?? '////'), '/')[4])
 }
 
-resource exampleResoruce 'Example.Resource/example@2023-01-31' = {
-  name: 'exampleResource'
+resource <exampleResource> 'Example.Resource/example@2023-01-31' = {
```

Instead of `exampleResource` is there any token we are using already in AVM to identify this? E.g. 
```suggestion
resource <singularResourceType> 'Resource.Provider/resourceType@2023-01-31' = {
```

---

Not that I'd be aware and I think it's anyways good enough if this schema is self-explanatory. Will update all examples

---

done @eriqua